ql/src/java/org/apache/hadoop/hive/ql/metadata/events/EventUtils.java

```diff
@@ -78,10 +79,15 @@ public long getCurrentNotificationEventId() throws IOException {
     }
 
     @Override
-    public long getDbNotificationEventsCount(long fromEventId, String dbName) throws IOException {
+    public long getDbNotificationEventsCount(long fromEventId, String dbName, Long toEventId,
+                                             int limit) throws IOException {
       try {
         NotificationEventsCountRequest rqst
                 = new NotificationEventsCountRequest(fromEventId, dbName);
+        if (toEventId != null)
```

Comment:
Need {} even for single line body for if clause.

Reply:
Done.